### DISCUSSION TITLE
Install location

### DISCUSSION THREAD

**Discussion by DakshNamdev** [coder/code-server#2376] [Q&A]
I'm trying to edit main login page for code-server, Where's the directory located?
I used the install script

**Comment by code-asher** [comment]
What was the output of the install script? The directory will depend on
what method it used to install code-server.

**Reply by DakshNamdev** [reply]
Um, I don't really know at this point it's been a week and I forgot it

**Reply by code-asher** [reply]
:P The install script will automatically choose a method based on your OS (assuming you didn't pass any flags to override this) so you can use that to figure out where it was installed: https://github.com/cdr/code-server/blob/master/doc/install.md#detection-reference

Another method would be to run

```
$ which code-server
```

To see where the entry script is and go from there to find the lib files. For example if it says `/usr/local/bin/code-server` then the files will be in `/usr/local/share/code-server`.

**Reply by DakshNamdev** [reply]
Yep, Found in /use/bin/Code-Server

Where's the CSS file I can edit? For login page

**Reply by code-asher** [reply]
I think it'll be `/usr/share/code-server/dist/main.css`.

**Reply by DakshNamdev** [reply]
![IMG_20201202_230621](https://user-images.githubusercontent.com/62242809/100909865-383ab180-34f3-11eb-8aa3-df9cc4f494c0.jpg)
I guess there's a issue?

**Reply by code-asher** [reply]
Oops I wrote `share` instead of `lib`. Is there a `/usr/lib/code-server`?

**Reply by DakshNamdev** [reply]
![IMG_20201202_235912](https://user-images.githubusercontent.com/62242809/100915512-78516280-34fa-11eb-9f7a-6e959c6c419e.jpg)
You can Dm me on discord if you have it we can talk further for the issue faster there I guess @Zorino#1110

**Reply by code-asher** [reply]
Ahh sorry I was confused, it's been renamed to `main.css` in a different branch that hasn't been merged yet. Currently all the CSS is bundled into `register.css` (not a great name which is why it'll get renamed soon).
